BlocSpot/BlocSpot/MapDisplayViewController.m

```diff
@@ -24,31 +25,33 @@ @implementation MapDisplayViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
-    self.locationManager = [[CLLocationManager alloc] init];
-    self.locationManager.delegate = self;
-    [self.locationManager requestWhenInUseAuthorization];
-    self.locationManager.desiredAccuracy = kCLLocationAccuracyBest;
+    self.searchTerm = @"Restaurants";
+    [[LocationManager sharedLocationManager] runLocationManager];
+}
+
+- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations {
```

Comment:
I added in the following method: 
- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations;
However, it doesn't seem to make a difference when it is being called.